module/VuFind/tests/integration-tests/src/VuFindTest/Mink/CartTest.php

```diff
@@ -270,7 +270,7 @@ protected function selectAllItemsInCart(Element $page)
     public function testAddingNothing()
     {
         // Activate the cart:
-        $this->changeConfigs(['config' => ['Site' => ['showBookBag' => true]]]);
+        $this->changeConfigs(['config' => ['Site' => ['showBookBag' => true, 'showBulkOptions' => true]]]);
```

Comment:
I understand now why you had to add showBulkOptions here to make the test pass -- it's because the cart button is being hidden because showTogglesInSearch is defaulting to true. If that really is the default that we want, it might be better to either adjust the tests to add things to the cart using the toggle button instead of the checkboxes, or else to set showTogglesInSearch to false in these tests, rather than setting showBulkOptions to true. It is only by a side effect that showBulkOptions is causing the cart controls to turn on, so I think this makes the test less clear than it could be.